Source/Core/DolphinWX/VideoConfigDiag.h

```diff
@@ -193,6 +193,8 @@ class VideoConfigDiag : public wxDialog
 		// Anti-aliasing
 		choice_aamode->Enable(vconfig.backend_info.AAModes.size() > 1);
 		text_aamode->Enable(vconfig.backend_info.AAModes.size() > 1);
+		if (vconfig.backend_info.bSupportsSSAA)
+			ssaa_checkbox->Enable(vconfig.iMultisampleMode > 0);
```

Comment:
Ah, yes, yes it does. Progamming in Java recently has spoilt me. </3 Is there a clean way of freeing the memory at the right time? Or would it be cleaner to simply create a greyed-out checkbox in Direct3D while SSAA gets implemented for that back-end?